docs/latest/gwchangev1.6.md

```diff
@@ -0,0 +1,60 @@
+---
+id: releaseNote1.6
+title: Godwoken V1.6 Release Note
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Godwoken is currently updated on a regular basis to provide new features and updates. The following release notes covers the most recent changes in Godwoken v1.5. You can check the full release notes in the [Godwoken Changlog](https://github.com/nervosnetwork/godwoken/blob/develop/CHANGELOG.md).
+
+The latest update to Godwoken v1.6 includes the following changes:
+
+- feat: add to_address to polyjuice parser in [#784](https://github.com/nervosnetwork/godwoken/pull/784)
+
+  Added `to_address` for checking and supporting native token transfers.
+
+- Support non EIP-155 tx in [#777](https://github.com/nervosnetwork/godwoken/pull/777)
+
+  Godwoken now supports `EIP-155` transactions. Due to the reuse of the chain_id field, if we deploy a different Godwoken implementation with `chain_id = 0`, then a valid `EIP-155` transaction cannot be sent on-chain because Godwoken would assume the transaction as a non-EIP-155 transaction. With this pr, Godwoken accepts layer-2 transactions with `chain_id = 0`. 
+
+- Validate native token transfer raw tx and signature in [#788](https://github.com/nervosnetwork/godwoken/pull/788)
+
+  Related to Polyjuice [#173](https://github.com/nervosnetwork/godwoken-polyjuice/pull/173), see Polyjuice updates below for details. 
+
+
+- refactor: remove sentry in [#780](https://github.com/nervosnetwork/godwoken/pull/780)
+
+- chore: bump version v1.5.0 in [#786](https://github.com/nervosnetwork/godwoken/pull/786)
```

Comment:
@Flouse Should we mention `bump version v1.5.0`  here?

Reply:
> @Flouse Should we mention `bump version v1.5.0` here?

modified